### ISSUE TITLE
Routes don't appear in proxy

### ISSUE THREAD

**Issue by SaadBazaz** [blomqma/next-rest-framework#76]
I've proxied my NextJS server (http://localhost:3000 -> **https://**localhost:3001) due to another package.

I can't see routes on the https route.
http, however, work fine.

**Comment by blomqma** [comment]
Could you provide some details on how your proxy is set up? Are you perhaps using the Next.js [rewrites](https://nextjs.org/docs/pages/api-reference/next-config-js/rewrites)?

**Comment by SaadBazaz** [comment]
My package.json:
```
"scripts": {
    "dev": "npx local-ssl-proxy --source 3001 --target 3000 & next dev",
    "build": "yarn generatespec && next build",
    "start": "next start",
    "lint": "next lint",
    "generateapi": "npx @rtk-query/codegen-openapi openapi-config.ts",
    "generatespec": "npx next-rest-framework generate && npx cpy openapi.json public/api"
  },
  ```

**Comment by blomqma** [comment]
Thanks, I managed to reproduce the issue and this should now be fixed in `v3.1.1`.
